annet/gen.py

```diff
@@ -266,6 +286,13 @@ def _old_new_per_device(ctx: OldNewDeviceContext, device: Device, filterer: Filt
     )
 
 
+def _update_json_config(json_files, file_name, new_config):
+    file = list(json_files[file_name])
+    file[0] = new_config
```

Comment:
Never empty?

Reply:
Yes, `new_json_fragment_files` always return files with first item of tuple.